src/twisted/web/http.py

```diff
@@ -2208,7 +2208,7 @@ class HTTPChannel(basic.LineReceiver, policies.TimeoutMixin):
     """
 
     maxHeaders = 500
-    totalHeadersSize = 16384
+    totalHeadersSize = 65536
```

Comment:
Minor comment.
Why not extract this into a variable ?

-------

And it looks like we have missing automates tests.

With `totalHeadersSize = 16384` in `trunk` branch, we should have a test that will check that a header size of `16384 + 1 ` will trigger a failure.

And with the change from this branch, I was expecting to see that test fail now.

Can you please look into writing a test to make sure that Twisted now accepts headers of size `65536` and rejects headers at `65536 + 1` ?

Thanks